messages/snapshot.create.json

```diff
@@ -1,13 +1,15 @@
 {
-  "description": "snapshot a scratch org",
```

Comment:
I never noticed this. I'd prefer: create a scratch org snapshot

Reply:
yeah, I changed it when I did the first review.